<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_4</class>
 <widget class="QDialog" name="fct_comp_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tickets d'un dossier et d'une catégorie</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Num dossier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_4_dossier">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Catégorie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4_categorie"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fct_4">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_4">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>noSpec</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateRep</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noPlace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noRang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateEmTick</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noDos</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_4">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_fct_4</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_4</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_4_dossier</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>fct_comp_4</receiver>
   <slot>refreshCatList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_4_categorie</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_4</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>refreshCatList()</slot>
 </slots>
</ui>
